compose.yml

```diff
@@ -90,25 +90,129 @@ services:
     env_file:
       - .secrets/label_studio_admin_password_pg.env
 
-  minio:
-    image: quay.io/minio/minio
-    container_name: minio
-    restart: always
-    volumes:
-      - ./minio:/data
+  postgres_authentik:
+    image: postgres:16.10
+    container_name: postgres_authentik
+    restart: unless-stopped
+    shm_size: 128 mb
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -d $${POSTGRES_DB} -U $${POSTGRES_USER}"]
+      start_period: 20s
+      interval: 30s
+      retries: 5
+      timeout: 5s
     environment:
-      MINIO_ROOT_USER: e4e_minio_admin
-      MINIO_ROOT_PASSWORD_FILE: /run/secrets/minio_admin_password
-    command: server /data --console-address ":9001"
+      POSTGRES_PASSWORD_FILE: /run/secrets/authentik_postgres_admin_password
+    volumes:
+      - ./authentik/postgres/data/:/var/lib/postgresql/data/:rw
+      - ./authentik/postgres/config/:/etc/postgresql/:ro
+      - ./authentik/postgres/scripts/:/docker-entrypoint-initdb.d/:ro
+      - /etc/passwd:/etc/passwd:ro
     secrets:
-      - minio_admin_password
+      - authentik_postgres_admin_password
+    command: --config_file=/etc/postgresql/postgres.conf
+    ports:
+      - 5434:5432
     networks:
+      - authentik
+    user: "${USER_ID}:${GROUP_ID}"
```

Comment:
What is USER_ID and GROUP_ID, is it possible for multiple users to be in the same group?

Reply:
This is the user/group the service runs as.  We want it to run as fabricant_admin.